README.md

```diff
@@ -391,6 +392,32 @@ For more examples and ideas, visit:
 
 ```
 
+#### Proxy Setup
+
+If the device is located behind a corporate proxy you may need to set the correct environment variables.
+This is done by configuring proxy behavior for dockerd in the daemon.json file as described in
+['Configure the Docker daemon to use a proxy server'][docker-proxy].
+
+The daemon.json file should be located at `/usr/local/packages/dockerdwrapper/localdata/daemon.json`
+on the device and should include the following properties:
+
+```json
+{
+  "proxies": {
+    "http-proxy": "http://proxy.example.com:3128",
+    "https-proxy": "https://proxy.example.com:3129",
+    "no-proxy": "*.test.example.com,.example.org,127.0.0.0/8"
+  }
+}
+```
+
+Setting the contents of the daemon.json file can be done either by adding it to the source code and
+rebuilding the application or by ssh:ing into the device with an already installed application and updating
```

Comment:
I think this sounds a bit better than "ssh:ing"
Reference: https://www.redhat.com/sysadmin/access-remote-systems-ssh 

```suggestion
rebuilding the application or by logging into the device over SSH with an already installed application and updating
```